{
    "GSD": {
        "alias": "CVE-2021-3461",
        "description": "A flaw was found in keycloak where keycloak may fail to logout user session if the logout request comes from external SAML identity provider and Principal Type is set to Attribute [Name].",
        "id": "GSD-2021-3461",
        "references": [
            "https://access.redhat.com/errata/RHSA-2021:2070",
            "https://access.redhat.com/errata/RHSA-2021:2065",
            "https://access.redhat.com/errata/RHSA-2021:2064",
            "https://access.redhat.com/errata/RHSA-2021:2063",
            "https://security.archlinux.org/CVE-2021-3461"
        ]
    },
    "namespaces": {
        "cve.org": {
            "CVE_data_meta": {
                "ASSIGNER": "secalert@redhat.com",
                "ID": "CVE-2021-3461",
                "STATE": "PUBLIC"
            },
            "affects": {
                "vendor": {
                    "vendor_data": [
                        {
                            "product": {
                                "product_data": [
                                    {
                                        "product_name": "keycloak",
                                        "version": {
                                            "version_data": [
                                                {
                                                    "version_value": "rh-sso7-keycloak 9.0.13"
                                                }
                                            ]
                                        }
                                    }
                                ]
                            },
                            "vendor_name": "n/a"
                        }
                    ]
                }
            },
            "data_format": "MITRE",
            "data_type": "CVE",
            "data_version": "4.0",
            "description": {
                "description_data": [
                    {
                        "lang": "eng",
                        "value": "A flaw was found in keycloak where keycloak may fail to logout user session if the logout request comes from external SAML identity provider and Principal Type is set to Attribute [Name]."
                    }
                ]
            },
            "problemtype": {
                "problemtype_data": [
                    {
                        "description": [
                            {
                                "lang": "eng",
                                "value": "CWE-613"
                            }
                        ]
                    }
                ]
            },
            "references": {
                "reference_data": [
                    {
                        "name": "https://bugzilla.redhat.com/show_bug.cgi?id=1941565",
                        "refsource": "MISC",
                        "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1941565"
                    }
                ]
            }
        },
        "nvd.nist.gov": {
            "configurations": {
                "CVE_data_version": "4.0",
                "nodes": []
            },
            "cve": {
                "CVE_data_meta": {
                    "ASSIGNER": "secalert@redhat.com",
                    "ID": "CVE-2021-3461"
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "en",
                            "value": "A flaw was found in keycloak where keycloak may fail to logout user session if the logout request comes from external SAML identity provider and Principal Type is set to Attribute [Name]."
                        }
                    ]
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": []
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "https://bugzilla.redhat.com/show_bug.cgi?id=1941565",
                            "refsource": "MISC",
                            "tags": [],
                            "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1941565"
                        }
                    ]
                }
            },
            "impact": {},
            "lastModifiedDate": "2022-04-02T00:00Z",
            "publishedDate": "2022-04-01T23:15Z"
        }
    }
}